chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1639,6 +1639,157 @@ impl pallet_services_payment::Config for Runtime {
     type WeightInfo = weights::pallet_services_payment::SubstrateWeight<Runtime>;
 }
 
+#[apply(derive_storage_traits)]
+#[derive(Copy, Serialize, Deserialize, MaxEncodedLen)]
+pub enum StreamPaymentAssetId {
+    Native,
+}
+
+pub struct StreamPaymentAssets;
+impl pallet_stream_payment::Assets<AccountId, StreamPaymentAssetId, Balance>
+    for StreamPaymentAssets
+{
+    fn transfer_deposit(
```

Comment:
I see the same in the dancebox, flashbox, we might want to consider re-using the the logic and put it to `client/node-common` probably.

Reply:
Created a ticket for common code (if possible)